remoteit/desktop
# [DESK-762] share user fix state check new services

### Description

Sharing a second user in the device incorrectly detecting shared-to state


### Test plan



1. Sign in
2. Go to a device
3. Click on share device
4. Select a user
5. Select a second user
4. You should see the combination of services and scripting 

### Ticket

https://remoteit.atlassian.net/browse/DESK-762
